src/screenComponents/radarView.cpp

```diff
@@ -86,7 +86,7 @@ GuiRadarView::GuiRadarView(GuiContainer* owner, string id, float distance, Targe
     show_waypoints(false),
     show_target_projection(false),
     show_missile_tubes(false),
-    show_callsigns(false),
```

Comment:
Don't understand why this default was changed? Seems like a change instead of a regression from stable?

Reply:
This is the code in screenComponents/radarView.cpp on 2024.12.08 that uses `show_callsigns` but is no longer present in `master`:

```
        if (obj != *my_spaceship && rect.overlaps(object_rect))
        {
            obj->drawOnRadar(renderer, object_position_on_screen, scale, view_rotation, long_range);
            if (show_callsigns && obj->getCallSign() != "" && obj->getDockedStyle() != DockStyle::Internal)
                renderer.drawText(sp::Rect(object_position_on_screen.x, object_position_on_screen.y - 15, 0, 0), obj->getCallSign(), sp::Alignment::Center, 15, bold_font);
        }
```

Reply:
Ok, then the display of callsigns on all radars could be a regression from legacy compared to master. I see legacy as leading for behavior, not master.

Reply:
OK, I'll revert to align to the legacy implementation.